packages/libs/package.json

```diff
@@ -44,7 +44,7 @@
   "dependencies": {
     "@audius/fixed-decimal": "0.1.1",
     "@audius/hedgehog": "3.0.0-alpha.1",
-    "@audius/sdk": "*",
+    "@audius/sdk": "6.0.1",
```

Comment:
probably don't actually need this pinned...

Reply:
if we publish this publicly, we do, so i think ok to keep 🤷